packages/i18nify-js/scripts/bundle-size-report.js

```diff
@@ -78,7 +78,7 @@ function differenceToSymbol(v) {
 }
 
 function traverseFileAndGenerateStats(dir) {
-  const filesToTrack = ['esm/index.js', 'cjs/index.js', 'umd/index.js'];
+  const filesToTrack = ['esm/index.min.js', 'cjs/index.js', 'umd/index.js'];
```

Comment:
Add comments to devs to understand, why we are doing this. REST is looks good.

Reply:
done